### PR TITLE
chore: add googleapis-dlp team as DLP product codeowners

### PR DESCRIPTION
**Pull request by annadata**
Please note that I see the following error. Probably @GoogleCloudPlatform/googleapis-dlp needs to be given write access to the repo.

Unknown owner on line 76: make sure the team @GoogleCloudPlatform/googleapis-dlp exists, is publicly visible, and has write access to the repository
…                              @GoogleCloudPlatform/googleapis-dlp

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -73,7 +73,7 @@
 # Does not have owner
 /asset/                                @GoogleCloudPlatform/go-samples-reviewers
 /auth/                                 @GoogleCloudPlatform/go-samples-reviewers
-/dlp/                                  @GoogleCloudPlatform/go-samples-reviewers
+/dlp/                                  @GoogleCloudPlatform/googleapis-dlp @GoogleCloudPlatform/go-samples-reviewers
```
Comment (telpirion) [inline .github/CODEOWNERS:76]: issue: it does not look like the googleapis-dlp group is part of the GoogleCloudPlatform organization. Please fix.

Comment (telpirion) [reply .github/CODEOWNERS:76]: nit: please keep @GoogleCloudPlatform/go-samples-reviewers as the first entry on the line.

**Review by telpirion — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (annadata) [reply .github/CODEOWNERS:76]: @telpirion Thanks for your review.
googleapis-dlp is being used for other doc samples repos.
I am replicating https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/2133/files done by @Sita04 recently. Kindly check if the error message included in description is relevant.

Question reg keeping  @GoogleCloudPlatform/go-samples-reviewers first: does that influence the order in which review is assigned?

Comment (Sita04) [reply .github/CODEOWNERS:76]: @telpirion [googleapi-dlp](https://github.com/orgs/GoogleCloudPlatform/teams/googleapis-dlp) exists. Should it be added to the repository permission to make the CODEOWNERS file valid?

**Review by telpirion — APPROVED** [review]

**Pull request merged**